mantis-control-plane/mantis-control-plane-server/dependencies.lock

```diff
@@ -5,138 +5,8 @@
         }
     },
     "baseline-exact-dependencies-main": {
-        "com.github.spullara.cli-parser:cli-parser": {
```

Comment:
Why was this file updated if there is no build.gradle changes?

Reply:
nice catch. interesting... not sure what happened. let me revert